uwblueprint/focus-on-nature
# feat: refund confirmation page API

## Notion ticket link

[Ticket Name](https://uwblueprintexecs.notion.site/Refund-Confirmation-API-20bc2f90b63b4e6194ac8a00d8723730)



## Implementation description
* Implemented get refund confirmation to get info of user wanting to get refund.
* Provided direct link access for server



## Steps to test
1. ensure that 404, 500, and 200 result paths are reached
2. ensure right data comes back



## What should reviewers focus on?
* ensure 200 path provides correct info


## Checklist
